src/pages/api/searchOperators.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiResponse } from 'next';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { SEARCH_OPERATOR_ATTRIBUTE } from '../../constants';
+import { redirectTo, redirectToError } from './apiUtils';
+import { updateSessionAttribute } from '../../utils/sessions';
+
+export interface SearchOperators {
+    searchResults: string[];
+}
+
+export interface SearchOperatorsWithErrors extends SearchOperators {
+    errors: ErrorInfo[];
+}
+
+export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
+    const errors: ErrorInfo[] = [];
+
+    try {
+        const { searchText } = req.body;
+
+        if (searchText.length < 4) {
```

Comment:
I think this is meant to be `.length < 3`

Reply:
can be <= 3 as well so will change